pkgs/development/compilers/abcl/default.nix

```diff
@@ -1,4 +1,4 @@
-{lib, stdenv, fetchurl, ant, jre, jdk}:
+{lib, stdenv, fetchurl, ant, jre, jdk, ...}:
```

Comment:
Is this `...` useful for anything or just a leftover?

Reply:
leftover - rolled it back [c2d2e82](https://github.com/NixOS/nixpkgs/pull/218870/commits/c2d2e8272acad408deb0c66a7b03d67b9f602dfe)